cibuildwheel/macos.py

```diff
@@ -82,6 +107,9 @@ def call(args, env=None, cwd=None, shell=False):
             installation_bin_path,
             env['PATH'],
         ])
+        # PyPy 3.6, v7.2.0 doesn't set MACOSX_DEPLOYMENT_TARGET in distutils.sysconfig, so let's set it ourselves until this gets settled
```

Comment:
To be fixed by https://github.com/pypa/wheel/pull/314 (see discussion 2 lines below)